### PR TITLE
Add lazy loading for 1995 form

### PR DESCRIPTION
**Pull request by jbalboni**
Related to #4608 

This converts the 1990 routes to the react router object format and loads the 1995 code lazily if the build type is `development`.

### CODE REVIEW

**Comment by jbalboni** [comment]
FYI @crwallace: we're lazy loading some parts of the edu benefits app now.

File: src/js/common/schemaform/helpers.js [hunk 1]
```diff
@@ -18,6 +16,19 @@ function createFormPageList(formConfig) {
     }, []);
 }
 
+export function createPageListByChapter(formConfig) {
```
Comment (U-DON) [inline src/js/common/schemaform/helpers.js]: Was confused that this is used in another PR, but not in this one, even though it was added here.

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/common/schemaform/helpers.js (hunk 1 above)
Comment (jbalboni) [inline src/js/common/schemaform/helpers.js]: Oh, oops. I'll remove it from here.

**Pull request merged**